### ISSUE TITLE
[admin] Make organization selection automatic when only 1 org

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-users#170]
When the user logged in has access to only one organization, the organization select should automatically set that org as the default value, the user should not need to select it.

**Comment by nemesisdesign** [comment]
**Update**:

this could do the trick: https://docs.djangoproject.com/en/dev/ref/contrib/admin/#django.contrib.admin.ModelAdmin.formfield_for_choice_field

```python
if len(request.user.organizations_managed) <= 1:
    kwargs['initial'] = request.user.organizations_maanged[0]
```

I have not tested this.

**Comment by pniaps** [comment]
I think that way would require code in all ModelAdmin where the model uses **organization** field.

I'm testing with the field, something like adding a default value to the field https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/mixins.py#L84

Let me think about your suggestion, I didn't knew about **organizations_managed** but its a good point.

**Comment by nemesisdesign** [comment]
> I think that way would require code in all ModelAdmin where the model uses **organization** field.

Not exactly. All the modules which support multitenancy inherit this mixin:
https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/multitenancy.py#L11

So it should be enough to implement it once for every multi-tenant admin.

**Comment by pniaps** [comment]
Great, I will try it.
Thanks.

**Comment by keshavgbpecdelhi** [comment]
Well done.

**Comment by nemesisdesign** [comment]
Implemented in https://github.com/openwisp/openwisp-users/issues/333.
